### PR TITLE
Allow fallback of Suspense to be mounted multiple times

### PR DESCRIPTION
**Pull request by sventschui**
I failed to reproduce #1813 in our demo app but with some debugging I came across the fact that the `_dom` property was missing on the fallback vnode. The reason for that seems to be the `coerceToVNode` in render since it clones the vnode. What I do not get is why I can not reproduce this in the demo app.

This PR adds 14B to preact and compat each.

Anybody has any ideas how to repro in demo?

TODO:
- [ ] Tests
- [ ] Revert demo

### CODE REVIEW

**Comment by sventschui** [comment]
Just as a short update. I found that the duplicate dom of some components can be prevented by adding a guard to _childDidSuspend that prevents tracking promises multiple times. It does not solve the stale fallback dom to still be present. 

Durinh these investigations I found further issues with dom nodes that are mounted as siblings to a suspending component. As a result of these issues I started investigating two-phase rendering.

TL;DR I looked into the issue and think Suspense is not possible with the current implementation of preact. The two phase render will take a lot of time to be built and will land post 10.0.0 release

**Comment by sventschui** [comment]
Closing this one as we need a two-phase diff algo to properly solve these issues
